taxonomies/additives.txt

```diff
@@ -4305,35 +4305,53 @@ efsa_evaluation:en: Scientific Opinion on the re-evaluation of propionic acid (E
 efsa_evaluation_adi_established:en: en:no
 additives_classes:en: en:preservative
 
-en:E282, Calcium propionate, Calcium propanoate
+# description:en:CALCIUM PROPIONATE has the formula Ca(C2H5COO)2. Calcium propionate is used as a preservative in a wide variety of products, including but not limited to:bread, other baked goods, processed meat, whey, and other dairy products.
+
+en:E282, calcium propionate
+ar:E282, بروبيونات الكالسيوم
+# azb:E282, کلسیوم پروپانوات
 bg:E282, Калциев пропионат
+ca:E282, Propanoat de calci
 cs:E282, Propionan vápenatý
 da:E282, Calciumpropionat
-de:E282, Calciumpropionat, E 282, Kalziumpropionat, Calciumdipropionat
+de:E282, Calciumpropionat
 el:E282, Προπιονικο ασβεστιο
-es:E282, Propionato cálcico, Propionato de calcio, Propanoato de calcio, Conservante E282, E-282, Conservante E 282, Propanoato cálcico
+eo:E282, kalcia propanato
+es:E282, Propionato de calcio, propionato cálcico
 et:E282, Kaltsiumpropionaat
-fi:E282, Kalsiumpropionaatti, Kalsiumpropanaatti
-fr:E282, Propionate de calcium, Propanoate de calcium
-hu:E282, Kalcium-propionát
-it:E282, Propionato di calcio
+fa:E282, کلسیم پروپانوات
+fi:E282, Kalsiumpropionaatti
+fr:E282, propionate de calcium
+hu:E282, kalcium-propionat
+it:E282, propionato di calcio
 lt:E282, Kalcio propionatas
 lv:E282, Kalcija propionāts
 mt:E282, Propjonat tal-kalċju
-nl:E282, Calciumpropionaat, Calciumpropanoaat
```

Comment:
Is removing Calciumpropanoaat intended? Same for German

Reply:
Yes, it is not used on products